aten/src/ATen/native/cuda/UnarySignKernels.cu

```diff
@@ -45,8 +45,22 @@ void sign_kernel_cuda(TensorIterator& iter){
   }
 }
 
+template<typename T>
+__host__ __device__ static inline c10::complex<T> sgn_wrapper(c10::complex<T> v) {
+  return v.sgn();
```

Comment:
Sorry, I'm not very familiar with c10::complex (yet), how does this work? Alternatively, where is this function implemented?